fastlane/spec/actions_specs/swiftlint_spec.rb

```diff
@@ -1,7 +1,7 @@
 describe Fastlane do
   describe Fastlane::FastFile do
     describe "SwiftLint" do
-      let(:swiftlint_gem_version) { Gem::Version.new('0.9.2') }
+      let(:swiftlint_gem_version) { Gem::Version.new('0.11.0') }
```

Comment:
Should you change this value?
This change may affect to exist tests.

I think it is better to override this value only for new test cases.

Reply:
I think so, by default I believe we should use the most recent version, but for tests, I set the latest version comparing with the other options. If I don't change it here, then I need to set the version for every test I made.